force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -218,14 +289,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createIntegerList(Integer n) {
         // Create a new list of integers 
-
+        List<Integer> integerList = new List<Integer>();
         /*
         for() {
             //add integer to the list
         }
         */ 
-
-        return null; // Replace null with the variable you used to store the result
+        for (Integer i = 1; i <=n; i++){
+            integerList.add(i);
+        }
+        return integerList; // Replace null with the variable you used to store the result
```

Comment:
good job. Try to get in the habit of deleting commented out code before it gets to Prod